In [1]:
import numpy as np 
import pandas as pd

In [2]:
data = pd.read_csv(r"C:\Users\ladm\Downloads\data 2.csv")

In [3]:
data.head()

2007   Jan.   Feb.   Mar.   Apr.   May.   Jun.  Jul.  \
0   Value of Pound in Euros  1.507  1.497  1.470  1.471  1.468  1.480  1.48   
1  Value of Pound in Dollar  1.959  1.959  1.947  1.988  1.984  1.987  2.03   

    Aug.   Sep.   Oct.   Nov.   Dec.  
0  1.476  1.451  1.437  1.411  1.384  
1  2.011  2.018  2.045  2.071  2.014

In [4]:
data

2007   Jan.   Feb.   Mar.   Apr.   May.   Jun.  Jul.  \
0   Value of Pound in Euros  1.507  1.497  1.470  1.471  1.468  1.480  1.48   
1  Value of Pound in Dollar  1.959  1.959  1.947  1.988  1.984  1.987  2.03   

    Aug.   Sep.   Oct.   Nov.   Dec.  
0  1.476  1.451  1.437  1.411  1.384  
1  2.011  2.018  2.045  2.071  2.014

## a. How many Dollars was one Euro worth in May?

In [5]:
#value of 1 Euro in dollars = Value of 1 Pound in Dollars in May / Value of 1 Pound in Euros in May
one_euro_dollar = data['May.'].iloc[1]/data['May.'].iloc[0]
one_euro_dollar

1.3514986376021796

## 1.35 Dollars was one Euro worth in May

## b.What was the percentage drop in the value of the Pound compared to the Euro over the year?

In [6]:
#Percentage change = (change in value / original value) * 100
percent = (data['Jan.'].iloc[0] - 
                  data['Dec.'].iloc[0])/data['Jan.'].iloc[0] * 100
print(percent)

8.161911081619097


## The percentage drop in the value of the Pound compared to the Euro over the year is 8.161911081619097 which roundoffs to 8.

## c. If you converted 67 Euros to Pounds in February, how many Dollars (to the nearest Dollar) would this be worth in November?

In [7]:
#67_euros_to_pound_in_feb = 67/ Value of Pound in Euros in feb
euros_to_pound = 67/data['Feb.'].iloc[0]
print(euros_to_pound)

44.756179024716104


In [8]:
# 44.75_pound_in_nov = 44.75 * Value of Pound in Dollar in nov
worth_in_nov = 44.75 * data['Nov.'].iloc[1]
print(worth_in_nov)

92.67724999999999


## So, it would be worth nearest to $93 in November .